### DISCUSSION TITLE
Just got an 5900x, would someone tell me a good settings for x265 please?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [staxrip/staxrip#675] [Q&A]
Till now i just used h265 cause my CPU was too slow (7700k), i know you will tell me to go and read each definition of each setting.
But i will like someone to give me a start point at least, with the most important (and not so evident) options i should use.

Will be used for:

- **Resolution:** 1080p.
- **Source:** Remuxes (most of them).
- **Audio:** I have my own custom preset already, not need help with the audio.

I know the most important point in this case, is to make use of the **"Chunks"** (3 or 4) to take advantage of the Cores/Threads.

**"Maximum number of parallel processes"**, should i keep it on **3** (default)?

Thx in advance.

**Comment by jessielw** [comment]
I suggest poking around the doom9 forums

**Comment by ghost** [comment]
@ZeroCool22 the `CHUNKS` must be always LESS OR EQUAL to `Maximum Number of Parallel Processes`


If you use `CHUNKS` = 3  I recommend you set `Maximum Number of Parallel Processes` = 4
If you use `CHUNKS` = 4  I recommend you set `Maximum Number of Parallel Processes` = 5

But, before using chunks, you have to do a "normal" (no chunk) encode, and see how much of your cpu is used. Let it encode like 30 minutes, and check on it every while.
- If you see your processor is around 30% then you'll know you'll need to use 3 chunks to take full advantage of your cpu
- If you see your processor is around 40% to 70% then you can use 2 chunks
- If your processor is above 80% then leave to 1 chunk.

Of course when doing this test, you must have no other load on the cpu, so you can know what is the load generated by the encoding.

And for your encoding settings, use x265, preset slow, and adjust the value of the crf (quality)

**Reply by jessielw** [reply]
Nice Vince, I didn't even know it worked like that

**Reply by ZeroCool22** [reply]
Thanks!

**Reply by ghost** [reply]
You're welcome :-)

**Comment by b6vertigo** [comment]
This guide is excellent for vapoursynth and has some good x264/x265 tips as well, https://silentaperture.gitlab.io/mdbook-guide/.

>  i know you will tell me to go and read each definition of each setting

The descriptions can help but x265 has around 300 settings and the docs often use very fancy wording (i.e. not newbie friendly) to describe what they do. My suggestion is to start with the presets and see where that takes you, and looking up options as you experiment. I never use any presets faster than 'Slower' with my 3800x but that's me. With a 5900x you could probably start with 'Very Slow,' even ramping some settings up to 'Placebo' or higher (--rc-lookahead 60, --subme 7) if you want to get crazy. It all depends on how much time/electricity you want to spend on compression/quality.
